stock_release_channel_shipment_lead_time/models/stock_release_channel.py

```diff
@@ -49,7 +51,13 @@ def _add_shipment_lead_time(self, dt):
         if not self.delivery_calendar_id:
             shipment_tz = fields.Datetime.add(dt_tz, days=self.shipment_lead_time)
         else:
-            days = self.shipment_lead_time + 1
+            days = self.shipment_lead_time
+            if days:
+                # We consider the current day as valid, we add a day, and then
+                # we look for open days for the given delay. Reset time to 0 in
+                # case the end time if after the warehouse calendar end time.
+                dt_tz += timedelta(days=1)
```

Comment:
You can avoid to import `timedelta`:
```suggestion
                dt_tz = fields.Datetime.add(dt_tz, days=1)
```

Reply:
I tried that first but it was not keeping the datetime in the localtime

Reply:
I retried and I have the same result :)

```
(Pdb) p dt_tz
datetime.datetime(2025, 1, 2, 9, 0, tzinfo=<DstTzInfo 'Europe/Brussels' CET+1:00:00 STD>)
(Pdb) p dt_tz + timedelta(days=1)
datetime.datetime(2025, 1, 3, 9, 0, tzinfo=<DstTzInfo 'Europe/Brussels' CET+1:00:00 STD>)
(Pdb) p fields.Datetime.add(dt_tz, days=1)
datetime.datetime(2025, 1, 3, 9, 0, tzinfo=<DstTzInfo 'Europe/Brussels' CET+1:00:00 STD>)
```

I'll do the change then